services/ore/ore-license.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const { test, forCases, given } = require('sazerac')
+const OreLicense = require('./ore-license.service')
+
+describe('OreLicense', function () {
+  test(OreLicense.prototype.transform, () => {
+    forCases([
```

Comment:
i don't think `forCases` is needed in these tests since there's just the one case right?